db/database_test.go

```diff
@@ -1656,3 +1656,24 @@ func BenchmarkDatabase(b *testing.B) {
 		db.Close()
 	}
 }
+
+func BenchmarkDatabase2(b *testing.B) {
```

Comment:
Is there a more appropriate name for what this test is doing? Maybe `BenchmarkPut` ?

Reply:
Very true... Intended to change that...